pkg/proxy/apis/config/validation/validation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	componentbaseconfig "k8s.io/component-base/config"
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
```

Comment:
this might complicate things for kubernetes/kubeadm#1600, I'd suggest dropping this import.

cc @neolit123 @rosti

Reply:
but this import is in pkg/proxy....validtion and not cmd/kubeadm.
kubeadm does not import pkg/features (last time i've checked). why do you see this as a problem?

Reply:
my bad, I thought I was reading on `cmd/kubeadm`

Reply:
@Arvinderpal - can you run `hack/update-bazel.sh` to reflect the introduced imports on the BUILD files ?

Reply:
Done!